game/src/main/java/tw/waterball/survive/io/game/gameloop/GameListener.java

```diff
@@ -0,0 +1,13 @@
+package tw.waterball.survive.io.game.gameloop;
+
+import tw.waterball.survive.io.game.sprite.Sprite;
+
+import java.util.List;
+
+/**
+ * @author - a89010531111@gmail.com
+ */
+public interface GameListener {
+    void onRender(List<Sprite> sprites);
```

Comment:
`onUpdate()`
GameLoop's Listener should not be bound to the concept of rendering